src/components/ShapeSelection.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react";
+
+const size = 70;
+
+class ShapeSelection extends React.Component<any, any> {
```

Comment:
it's always a good idea to define a props interface. this way you get compile time checking for the component.